app/controllers/api/v0/mixins/index_mixin.rb

```diff
@@ -3,7 +3,17 @@ module V0
     module Mixins
       module IndexMixin
         def index
-          render json: model.where(list_params)
+          render json: scoped(model.where(list_params))
+        end
+
+        def scoped(relation)
+          if model.respond_to?(:taggable?) && model.taggable?
+            ref_schema = {model.tagging_relation_name => :tag}
+
+            relation = relation.includes(ref_schema).references(ref_schema)
+          end
```

Comment:
Minor but I wonder if we can move this to the tagging mixin in core?

Reply:
I guess that could work

Reply:
Would be a good follow-up PR, then we can drop this from the two mixins here